package fly2us.tn.user.dto;

import lombok.Data;

@Data
public class LoginRequest {
    private String email;
    private String password;
}